tests/utils.py

```diff
@@ -17,16 +17,12 @@ def compare_charts(actual: List[str], expected: List[str]):
 
 
 def compare_chart_and_markdown(chart: str, markdown_path: str):
-    actual = chart.split("\n")
-    actual.sort()
+    markdown_path = Path(markdown_path)
```

Comment:
I added the output of the generated files, which makes it easier to see what happen if a test failed. With this, the compare_chart_and_markdown can be reduced to the compare_markdown function.